### PR TITLE
Setup and controllers

### PR DESCRIPTION
**Pull request by Kakalanp**
# Changes in this PR:

I've added routes, actions, controllers, and views.

## Additions:
- routes:
  - users
  - users/`id`
  - users/`id`/posts
  - users/`id`/posts/`id`
- controllers:
  - posts
  - users
- views:
  - users:
    - index
    - show
  - posts
    - index
    - show
- extras:
  - linters:
    - stylelint
    - rubocop

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :users, path: '/users' do
+    resources :posts, path: '/posts'
+  end
```
Comment (andersonlebon) [inline config/routes.rb]: ## [OPTIONAL]

- [x] Kindy I would suggest you make sure your app has a default root that will be used when the application starts.

```rb
  root 'users#index'
```

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Kakalanp 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly make sure there you have a professional readme file that contains all the sections from [the microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md)

image


- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (Kakalanp) [reply config/routes.rb]: I didn't know how to do that! Thank you very much :+1:

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Kakalanp 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the required changes 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kakalanp** [comment]
Thank you!

**Pull request merged**